package.json

```diff
@@ -108,7 +109,9 @@
     "nyc": "^15.0.1",
     "prettier": "^2.0.4",
     "tmp": "^0.1.0",
+    "type-fest": "^0.13.1",
     "ts-node": "^8.10.1",
-    "tslint": "^6.1.2"
+    "tslint": "^6.1.2",
+    "xo": "^0.30.0"
```

Comment:
Don't use xo, use Google's tool "gts" since this is a Google project.

Reply:
I'll look into `gts` and compare with `xo`

Having Google style enforced by `gts`, `xo` or even a combination of both should not an issue.

Depending on how good `gts` is at linting, I may keep `xo` as secondary linter for advanced feature.

> since this is a Google project.

Google should ensure some gouvernance here imho

Reply:
@grant about `gts`

- nice npm scripts
- the default `tsconfig.json` is cute but with little added value compared to the current one (i.e. we can setup to `extends` from the `gts` defaults and then add a few specific options.)
- the Prettier config differs only for `bracketSpacing` and `arrowParens`. There again we should extends the default `gts` with only distinct settings
- the Eslint configuration is minimalist (imho). I haven't fully reviewed the ruleset yet. I could probably extend it so that it takes advantages of `xo` ruleset (while of course maintaining `gts` prevalence)

Anyhow, `gts` implies some additional (and sound) file management, mostly by having compiled .js files under `build`, which I had on my onw todo list.

Please advise.

Reply:
If you want a strict answer, then use `gts`:
https://github.com/google/gts
This is a Google project. Although there is no current maintainer, we must use Google conventions.

Follow the tooling in this repo and no more (no eslint, no xo):
https://github.com/googleapis/google-api-nodejs-client/blob/master/package.json

Reply:
Ok. I'll cleanup the extra config when done with the refactoring.

Somewhat a pity for current config follows `gts` requirements but adds stricter rules which improve code quality (imho)